### ISSUE TITLE
item pictures with configuration variants

### ISSUE THREAD

**Issue by kurthatlevik-inspirit365** [MicrosoftDocs/dynamics-365-unified-operations-public#3446]
Hi.

In the documentation we see :"Product variant images should be named "/Products/{ProductNumber} ^ {Style} ^ {Size} ^ {Color} ^_000_001.png""

But I have customers that are using configuration, and would like to a configuration spesific product pictures.  What are the naming convension when having configuration ?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6d8c889e-fd24-912c-61bd-40abaa7a18ca
* Version Independent ID: 79ac40fe-2a63-07f0-24ac-4c61d56d25ef
* Content: [Upload images - Commerce | Dynamics 365](https://docs.microsoft.com/en-us/dynamics365/commerce/dam-upload-images)
* Content Source: [articles/commerce/dam-upload-images.md](https://github.com/MicrosoftDocs/Dynamics-365-Unified-Operations-Public/blob/main/articles/commerce/dam-upload-images.md)
* Service: **dynamics-365-commerce**
* Product: ****
* Technology: ****
* GitHub Login: @psimolin
* Microsoft Alias: **psimolin**

**Comment by RobinARH** [comment]
@psimolin - Can you take a look? Thanks.

**Comment by RobinARH** [comment]
Created bug 602394 to track this issue.

**Comment by kurthatlevik-inspirit365** [comment]
> Hi.
> 
> In the documentation we see :"Product variant images should be named "/Products/{ProductNumber} ^ {Style} ^ {Size} ^ {Color} ^_000_001.png""
> 
> But I have customers that are using configuration, and would like to a configuration spesific product pictures. What are the naming convension when having configuration ?
> 
> #### Document Details
> ⚠ _Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking._
> 
> * ID: 6d8c889e-fd24-912c-61bd-40abaa7a18ca
> * Version Independent ID: 79ac40fe-2a63-07f0-24ac-4c61d56d25ef
> * Content: [Upload images - Commerce | Dynamics 365](https://docs.microsoft.com/en-us/dynamics365/commerce/dam-upload-images)
> * Content Source: [articles/commerce/dam-upload-images.md](https://github.com/MicrosoftDocs/Dynamics-365-Unified-Operations-Public/blob/main/articles/commerce/dam-upload-images.md)
> * Service: **dynamics-365-commerce**
> * Product: ****
> * Technology: ****
> * GitHub Login: @psimolin
> * Microsoft Alias: **psimolin**

See also "Images for product variants #3495", that shows a way how product variants and can different pictures

**Comment by oklinge** [comment]
Just found out how this is done. I created a service request on the topic and was told that only Style, Size and Color dimensions were supported. Found this to be wrong after going through the development tools console error messages.

For an item that has configuration dimension I found a working format to be as follows:

[Product number] ^ [Configuration]_000_001.png

Product number is the item product number from D365FO _All Products and products master_, and the Configuration is the column _Configuration_ under the menu item Product dimensions.

![image](https://user-images.githubusercontent.com/80251989/128341399-1c21c876-2979-4bc1-b764-5eaaced16db6.png)

Example
108119 ^ LB8017_000_001.png

File name with all characters displayed.
![image](https://user-images.githubusercontent.com/80251989/128341937-9a43c72e-3c24-4342-871c-602c39cb3abb.png)
